patches/cac@6.7.14.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/dist/index.mjs b/dist/index.mjs
+index 7c1c4440c6ed22a3829f828364b06669c56b31c9..f464611b5b98c2935c2d92641dfe628f161ebca4 100644
+--- a/dist/index.mjs
++++ b/dist/index.mjs
+@@ -240,9 +240,10 @@ class Option {
+       return camelcaseOptionName(name);
+     }).sort((a, b) => a.length > b.length ? 1 : -1);
+     this.name = this.names[this.names.length - 1];
+-    if (this.negated && this.config.default == null) {
+-      this.config.default = true;
+-    }
++    // disable automatic setting of the defaults for negated options
```

Comment:
I'm not sure about this hack. This is needed to remove setting of the automatic default for negated option described in the [cac docs](https://github.com/cacjs/cac#negated-options).

Reply:
we can't patch cac because it's not a `devDependency`. 

Rethinking these cli options: Maybe we can just rely on the env vars instead of having cli options? 

Or is this something you have a use case for?

Reply:
I think CLI options is good and it does simplify a lot of stuff but I don't understand the purpose of this one?

Reply:
@ds300 I think it'd be good to keep CLI options as well as allow setting it as env var.

Basically, using help option in non-patched version outputs following:
```
  --color           forces the use of color in the output 
  --no-color        suppresses the use of color in the output (default: true)
```
However, it doesn't set `--no-color` to true and we still have colors in the output by default. My concern here is confusing help message that implies that there won't be colors by default.

The only workaround I can think of is postprocessing help message and removing this confusing bit of text, though it's quite hacky solution.